assets/src/settings-page/template-modes.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * External dependencies
+ */
+import {
+	IS_CORE_THEME,
+	THEME_SUPPORT_ARGS,
+	THEME_SUPPORTS_READER_MODE,
+} from 'amp-settings';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useContext } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { TemplateModeOption } from '../components/template-mode-option';
+import '../css/template-mode-selection.css';
+import { AMPNotice, NOTICE_SIZE_LARGE, NOTICE_TYPE_INFO, NOTICE_TYPE_WARNING } from '../components/amp-notice';
+import { Options } from '../components/options-context-provider';
+
+/**
+ * Provides the notice to show in the reader theme support mode selection.
+ *
+ * @param {string} themeSupport The current theme support mode.
+ */
+function getReaderNotice( themeSupport ) {
+	switch ( true ) {
+		// Theme has built-in support or has declared theme support with the paired flag set to false.
+		case 'reader' === themeSupport && ( 'object' === typeof THEME_SUPPORT_ARGS && false === THEME_SUPPORT_ARGS.paired ):
+			return (
+				<AMPNotice size={ NOTICE_SIZE_LARGE } type={ NOTICE_TYPE_WARNING }>
```

Comment:
`NOTICE_TYPE_WARNING` looks like an error to me. Shouldn't it rather be orange/yellow/

![image](https://user-images.githubusercontent.com/134745/87075366-11db4900-c1d5-11ea-940c-654cece32706.png)

Reply:
@jwold Can you recommend a color and maybe an alternative icon for this?

Reply:
Could we do something like this? https://d.pr/i/8gpdrm

SVG: https://d.pr/f/X2wkPh

Reply:
@jwold That's the info notice, which we're using in a couple of places on the same screen to make gentle suggestions. In this particular case the user has clicked Reader mode even though they shouldn't, so we want to warn them it's probably not the right choice.

Reply:
Oh good point. This might work for now:

<img width="989" alt="Screen Shot 2020-07-14 at 8 53 23 AM" src="https://user-images.githubusercontent.com/3411173/87447464-7d476100-c5af-11ea-8c12-8ef3f4e419b4.png">